analysis/_site.yml

```diff
@@ -1,25 +1,28 @@
 name: fucci-seq
-output_dir: "../docs"
-include: [".nojekyll"]
-exclude: ["include"]
+output_dir: ../docs
+include: .nojekyll
+exclude: include
 navbar:
```

Comment:
Why did you change this? Was it causing problems? Did you confirm it still works as expected?

Reply:
I previewed index.Rmd and it worked as expected. I didn't change these. After I finished knitting the two Rmds that I want to commit, I saw these files changed when I do `git status`... Not sure at which point of the process that these were generated.

Reply:
What version of rmarkdown are you running? `packageVersion("rmarkdown")` Maybe editing `_site.yml` is a newer feature?

Reply:
it's 1.8

Reply:
Strange. I didn't see anything in the [NEWS file](https://github.com/rstudio/rmarkdown/blob/master/inst/NEWS) to indicate this change. Not sure what caused this.